### DISCUSSION TITLE
Vercel deployed react app can not make API calls

### DISCUSSION THREAD

**Discussion by krishnasapkal** [vercel/community#2455] [Help]
### Summary

Getting below error for every http request react app makes after deploying with vercel

![image](https://github.com/vercel/community/assets/47318894/8e2de3b9-67f1-4499-9dcd-10408126f3e4)

**Comment by amyegan** [comment]
It looks like you have JavaScript disabled in your browser. In Chrome, you can change it in `Settings` > `Privacy & Security` > `JavaScript`. You'll need to make sure that you either allow all sites to use JavaScript or add your site to the allow list. In Firefox, you can toggle `javascript.enabled` to `false` on the about:config page.

It's also possible you disabled JavaScript with Chrome DevTools for testing. You can re-enable it following these steps: https://www.howtogeek.com/663569/how-to-disable-and-enable-javascript-on-google-chrome/#:~:text=Disable%20JavaScript%20with%20Chrome%20DevTools%20for%20Testing

**Reply by krishnasapkal** [reply]
> It looks like you have JavaScript disabled in your browser. In Chrome, you can change it in `Settings` > `Privacy & Security` > `JavaScript`. You'll need to make sure that you either allow all sites to use JavaScript or add your site to the allow list. In Firefox, you can toggle `javascript.enabled` to `false` on the about:config page.
> 
> It's also possible you disabled JavaScript with Chrome DevTools for testing. You can re-enable it following these steps: https://www.howtogeek.com/663569/how-to-disable-and-enable-javascript-on-google-chrome/#:~:text=Disable%20JavaScript%20with%20Chrome%20DevTools%20for%20Testing

Thanks @amyegan for the answer.
its happening on all of the browsers. I have checked JavaScript is enabled. could it be the way Vercel is serving the static files ?
